File: Examples/DotNET/SDK/VisualBasic/Document/AcceptTrackingChanges.vb
Imports Aspose.Cloud

Namespace Document
    Class AcceptTrackingChanges
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample.doc"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, String.Empty)

            ' Accept All Tracking Changes in the Document
            Dim wordRevisionResponse As WordRevisionResponse = Common.WordsService.WordsRevisions.AcceptAllRevisionsInDocument(inputfile, Common.GetOutputFilePath(inputfile, True), Common.FOLDER)

            Console.WriteLine(Convert.ToString(vbLf & "All tracking changes accepted in ") & inputfile)
        End Sub
    End Class
End Namespace

File: Examples/DotNET/SDK/CSharp/Document/AcceptTrackingChanges.cs
using System;
using Aspose.Cloud;

namespace CSharp.Document
{
    class AcceptTrackingChanges
    {
        public static void Run()
        {
            string inputfile = "doc-sample.doc";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, string.Empty);
            
            // Accept All Tracking Changes in the Document
            WordRevisionResponse wordRevisionResponse = Common.WordsService.WordsRevisions.AcceptAllRevisionsInDocument(inputfile, Common.GetOutputFilePath(inputfile,true), Common.FOLDER);

            Console.WriteLine("\nAll tracking changes accepted in " + inputfile);
        }
    }
}
